test-requirements.txt

```diff
@@ -4,3 +4,4 @@ fastapi
 flask
 psycopg2
 requests
+urllib3
```

Comment:
no urllib?

Reply:
Nope, urllib is part of the python standard library https://docs.python.org/3/library/urllib.html